lib/search.js

```diff
@@ -46,6 +34,9 @@ function search (args, silent, staleness, cb) {
 
   var searchopts = npm.config.get('searchopts')
   var searchexclude = npm.config.get('searchexclude')
+  if (npm.config.get('searchsort', 'cli')) {
+    log.warn('search', 'Search result sorting not currently supported')
+  }
```

Comment:
@legodude17 not for this iteration, and since this PR is a stopgap to get search working *at all*, it's unlikely that we'll have more featureful searching until we get better backend support for fast searching.